docs/pages/reference/access-controls/roles.mdx

```diff
@@ -405,7 +404,165 @@ spec:
+| `session.kubernetes_cluster` | The name of the Kubernetes cluster where the session took place. | `string` |
+| `session.kubernetes_labels` | The labels assigned to the Kubernetes cluster where the session took place. | `dict<string, string>` |
+| `session.kubernetes_user` | The Kubernetes user used in the session. | `string` |
+| `session.kubernetes_groups` | The Kubernetes groups used in the session. | `array<string>` |
+| `session.kubernetes_pod_namespace` | The namespace in the Kubernetes cluster where the session took place. | `string` |
+| `session.kubernetes_pod_name` | The name of the pod in the Kubernetes cluster where the session took place. | `string` |
+| `session.kubernetes_container_name` | The name of the container in the Kubernetes cluster where the session took place. | `string` |
+
+</TabItem>
+<TabItem value="Database Sessions" label="Database Sessions">
+
+The following database session fields can be used in `where` expressions to filter access
+for database sessions together with the user and session fields above.
+
+| Database Session Field | Description | Type |
+|---|---|---|
+| `session.db_service` | The name of the database service where the session took place. | `string` |
+| `session.db_protocol` | The protocol used to connect to the database (e.g., "postgres", "mysql"). | `string` |
+| `session.db_uri` | The URI of the database where the session took place. | `string` |
+| `session.db_name` | The name of the database accessed during the session. | `string` |
+| `session.db_user` | The database user used during the session. | `string` |
+| `session.db_labels` | The labels assigned to the database where the session took place. | `dict<string, string>` |
+| `session.db_type` | The type of the database (e.g., "postgres", "mysql"). | `string` |
+
+</TabItem>
+<TabItem value="Windows Desktop Sessions" label="Windows Desktop Sessions">
+
+The following Windows Desktop session fields can be used in `where` expressions to filter access
+for Windows Desktop sessions together with the user and session fields above.
+
+| Windows Desktop Session Field | Description | Type |
+|---|---|---|
+| `session.windows_desktop_service` | The name of the Windows Desktop service where the session took place. | `string` |
+| `session.desktop_addr` | The address of the Windows desktop where the session took place. | `string` |
+| `session.desktop_name` | The name of the Windows desktop where the session took place. | `string` |
+| `session.domain` | The domain of the Windows desktop where the session took place. | `string` |
+| `session.windows_user` | The username used to log in to the Windows desktop during the session. | `string` |
+| `session.desktop_labels` | The labels assigned to the Windows desktop where the session took place. | `dict<string, string>` |
+
+</TabItem>
+</Tabs>
+
+### Functions
+
+You can use the following functions in `where` expressions to filter access to
+session recordings:
+
+| Function | Signature |Description |
```

Comment:
Is this the full list of functions, or does session recording RBAC only support a subset of them?

I'm just thinking about future maintainability - how will we remember to update this list when we add support for new functions? Maybe some sort of centralized list would help?